### DISCUSSION TITLE
Make 'use bindings' persist for testing

### DISCUSSION THREAD

**Discussion by Nasnl** [Budibase/budibase#5892] [Ideas]
In automation:

I'm not sure if it is intended to work like this, but I think it would be a good idea to keep the 'use bindings' setting persistent/saved after you test.
In the current situation the setting is default 'use values' and even if you set it to 'use bindings', then you have to remember to do that again for each test, because it resets to default every test.

**Comment by melohagan** [comment]
Hey @Nasnl 

This is a bug, which I've raised here: https://github.com/Budibase/budibase/issues/5910

**Reply by Nasnl** [reply]
Txs for checking @melohagan !
